lifecycle-operator/chart/templates/lifecycle-mutating-webhook-configuration.yaml

```diff
@@ -34,6 +35,17 @@ webhooks:
       - keptn-lifecycle-toolkit-system
       - observability
       - monitoring
+{{- else }}
+    - key: kubernetes.io/metadata.name
+      operator: In
+      values: {{ .Values.allowedNamespaces | default list | toJson }}
+    - key: kubernetes.io/metadata.name
+      operator: NotIn
+      values:
+      - kube-system
```

Comment:
instead of repeating these kube-api rules, I would put them outside of the if and cleanup the other NotIn rule. This should work since all selector requirements are computed in a logical AND

Reply:
resolved